go.mod

```diff
@@ -58,3 +58,5 @@ require (
 	golang.org/x/oauth2 v0.0.0-20210220000619-9bb904979d93 // indirect
 	google.golang.org/genproto v0.0.0-20210222212404-3e1e516060db // indirect
 )
+
+replace github.com/hashicorp/hcat => /Users/mwilkerson/dev/hcat
```

Comment:
I believe an alternative way you can do this is to set it to the commit sha of your changes in hcat since they're pushed to the repo. You'd still need to update it to the proper version eventually, but then at least CircleCI can run with your changes. Disclaimer is I haven't tried this myself, but I think I've seen this done by others when reviewing PRs!

Reply:
oh yes, that makes sense, I'll give it a shot